support/values.yaml

```diff
@@ -71,3 +69,26 @@ grafana:
           access: proxy
           isDefault: true
           editable: false
+
+# Values reference for nfs-server-provisioner
+# https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner/blob/master/deploy/helm/values.yaml
+nfs-server-provisioner:
+  enabled: false
+  storageClass:
+    # Users will have to opt-in to using NFS-type storage under
+    # singleuser.storage.dynamic.storageClass
+    name: nfs
+    create: true
+    defaultClass: false
+    mountOptions:
+      - vers=4.2
+      - soft
+      - noatime
+  persistence:
+    enabled: true
+    accessMode: ReadWriteOnce
+    size: 100Gi
+    # Future option is to reference an xfs storage class. This will allow us to enable quotas.
+    # https://github.com/pangeo-data/pangeo-cloud-federation/issues/654#issuecomment-861771398
```

Comment:
On most cloud providers, the storageClass is a reference to the underlying *physical* storage only - spinning disks or SSDs. I think the nfs external provisioner already uses XFS, and so the storageClass wouldn't come up. I think the reference in the provided link is for bare metal instances only.